package.json

```diff
@@ -1,37 +1,33 @@
 {
-  "name": "typescript-lc39-hello-world-example",
+  "name": "mia_template_service_name_placeholder",
   "version": "0.1.0",
-  "description": "Hello World Example with typescript and LC39",
-  "main": "lib/index.js",
+  "description": "%CUSTOM_PLUGIN_SERVICE_DESCRIPTION%",
+  "main": "dist/index.js",
   "scripts": {
     "checkonly": "! grep -R '\\.only' tests/",
     "coverage": "npm run unit -- --coverage-report=text-summary",
     "postcoverage": "tap --coverage-report=lcov --no-browser",
     "lint": "eslint . --ignore-path=.gitignore",
-    "start": "lc39 lib/index.js",
+    "start": "lc39 dist/index.js",
     "start:local": "npm start -- --env-path ./default.env",
     "test": "npm run lint && npm run unit && npm run checkonly",
     "update-docker-version": "sed -i.bck \"s|version=\\\"[0-9]*\\.[0-9]*\\.[0-9]*.*\\\"|version=\\\"${npm_package_version}\\\"|\" Dockerfile",
     "update-changelog": "node ./scripts/update-changelog.js ${npm_package_version}",
     "unit": "tap -o tap.log tests/*.test.js",
     "version": "npm run update-changelog && npm run update-docker-version && rm -fr Dockerfile.bck && git add CHANGELOG.md Dockerfile",
-    "typescript": "tsc --project tsconfig.json"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/mia-platform-marketplace/Typescript-LC39-Hello-World-Example.git"
+    "build": "tsc --project tsconfig.json"
   },
   "license": "SEE LICENSE IN LICENSE",
   "dependencies": {
     "@mia-platform/custom-plugin-lib": "^1.1.1",
-    "fastify": "^2.14.1"
+    "fastify": "^2.14.1",
+    "typescript": "^3.9.5"
```

Comment:
This should be placed in the `devDependencies`

Reply:
Since NODE_ENV=production, `devDependencies` don't get installed and we can't build with Typescript if we place it among the dev dependencies.

Reply:
Done